java/base-client/src/main/java/no/bankaxept/epayment/client/base/http/HttpResponse.java

```diff
@@ -16,6 +18,15 @@ public HttpStatus getStatus() {
     return status;
   }
 
+  public RequestStatus requestStatus() {
+    if (status.code() == 200) return RequestStatus.Repeated;
+    else if (status.code() == 201) return RequestStatus.Accepted;
+    else if (status.code() == 409) return RequestStatus.Conflicted;
+    else if (status.code() == 422) return RequestStatus.Rejected;
+    else if (status.is4xxClientError()) throw new ClientError(body);
+    else return RequestStatus.Failed;
```

Comment:
Is it OK to not provide a failure reason on 5xx status codes?

Reply:
I think so. The client can't use that information for anything.